code/unclassified/leap_year/leap_year.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+
+bool isLeapYear(int year)
+{
+    if (year % 400 == 0)
+        return true;
+
+    if (year % 100 == 0)
+        return false;
+
+    if (year % 4 == 0)
+        return true;
+    return false;
+}
+
+void findLeapYears(int a , int b)
+{
+    cout<<"The leap years between "<<a<<" and "<<b<<" are : \n";
+    for(int i=a ; i<=b ; i++)
```

Comment:
add spaces (i.e. in here replace this with `for (int i = a; i <= b; i++)`, and all of conditions and operators. check this [coding styles for c++](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c%2B%2B)